### ISSUE TITLE
Import misses objects with postgresql

### ISSUE THREAD

**Issue by pmall** [laravel/scout#104]
Using postgresql, when I import some model, the order they are imported is totally messed up : 

~~~
php artisan scout:import "App\Domain\Method"
Imported [App\Domain\Method] models up to ID: 101
Imported [App\Domain\Method] models up to ID: 234
Imported [App\Domain\Method] models up to ID: 369
Imported [App\Domain\Method] models up to ID: 500
Imported [App\Domain\Method] models up to ID: 630
Imported [App\Domain\Method] models up to ID: 761
Imported [App\Domain\Method] models up to ID: 895
Imported [App\Domain\Method] models up to ID: 1024
Imported [App\Domain\Method] models up to ID: 1155
Imported [App\Domain\Method] models up to ID: 1284
Imported [App\Domain\Method] models up to ID: 489
Imported [App\Domain\Method] models up to ID: 953
Imported [App\Domain\Method] models up to ID: 1
~~~

This results in some models not being imported.

To work this around I have to overwrite the makeAllSearchable function, which is not cool : 

~~~
public static function makeAllSearchable()
{
    $model = new static;

    $model->newQuery()->orderBy($model->getKeyName(), 'ASC')->searchable();
}
~~~

**Comment by pmatseykanets** [comment]
@pmall Thanks for bringing this up. I wanted to make this PR for quite some time while working on [laravel-scout-postgres](https://github.com/pmatseykanets/laravel-scout-postgres). Taylor just merged #105 and tagged [v1.1.8](https://github.com/laravel/scout/releases/tag/v1.1.8). Can you try it to see if it works for you?

**Comment by pmall** [comment]
@pmatseykanets This work fine with v1.1.8. By the way your postgres package is great. Any plans to integrate trigram indexes? I mostly do autocompletion on large tables.
